### DISCUSSION TITLE
Upcoming XCP-ng 8.3 release

### DISCUSSION THREAD

**Discussion by ydirson** [apache/cloudstack#8964] [General]
We're going to release XCP-ng 8.3 in the upcoming months.  Is there anything particular we can do to ensure it will work properly with Cloudstack ?

**Comment by sbrueseke** [comment]
Are there any api changes? As far as I understand version 8.3 should work as long as the used api calls are still working.

**Reply by ydirson** [reply]
No API breakage, only introduction of new APIs for new functionalities (e.g. vTPM)

**Comment by shwstppr** [comment]
We may need to add some changes for hypervisor support, similar to https://github.com/apache/cloudstack/pull/4672 (maybe part of it).
But this will be done after release in my opinion.

**Comment by yadvr** [comment]
Good to hear! It's been a while since xcp-ng had a release. If it is backward compatible wrt API, usage and guest OS then adding support would be a straightforward entry in the hypervisor support related db tables.

**Reply by DaanHoogland** [reply]
if this is really all #10470 should fix this.

**Comment by ydirson** [comment]
Hello,

After publishing a new RC, we're currently planning to make the final release for September 30th.

**Comment by ydirson** [comment]
The 8.3 release [is now out](https://docs.xcp-ng.org/releases/release-8-3/) (official announcement will follow in a few hours).

**Comment by UAnton** [comment]
Any news?

**Comment by weizhouapache** [comment]
@UAnton 
please keep eye on #10483
